Release.toml

```diff
@@ -29,3 +29,7 @@ version = "1.0.6"
     "migrate_v1.0.6_add-shibaken.lz4",
     "migrate_v1.0.6_admin-container-v0-6-0.lz4",
 ]
+"(1.0.6, 1.1.0)" = [
+    "migrate_v1.1.0_kubelet-kube-reserved-and-eviction-hard.lz4",
```

Comment:
Other than not having a few extra migrations - is there any reason to bundle these settings into a single migration?

Reply:
The reason why I bundle them is that the settings which are bundled into one migration use same `migration`. For example. `settings.kubernetes.kube-reserved` and `settings.kubernetes.eviction-hard` use `AddPrefixesMigration`. I think that could be much easier to review and maintain without losing functionality.